Gems/Atom/RPI/Code/Source/RPI.Reflect/Shader/ShaderOptionGroupLayout.cpp

```diff
@@ -85,14 +85,16 @@ namespace AZ
             if (auto* serializeContext = azrtti_cast<SerializeContext*>(context))
             {
                 serializeContext->Class<ShaderOptionDescriptor>()
-                    ->Version(4)
+                    ->Version(5)
```

Comment:
Let's add a quick comment to induce the changes going with this version. We do this in a bunch of other places too.